City-of-Helsinki/drupal-asuntotuotanto
# SATT-68: Drupal core 10.3 update and helfi_platform config update

> [!IMPORTANT]  
> Crucial information necessary for users to succeed.
> Make sure to include https://github.com/City-of-Helsinki/drupal-asuntotuotanto/pull/579 in this upgrade.
> helfi_api_base 2.7.6 https://github.com/City-of-Helsinki/drupal-module-helfi-api-base/releases/tag/2.7.6
### Description

Drupal 10.3 update and helfi platform configure update

### How to test 

- make fresh or make up build sync drush-deploy
- login as admin
- rebuild tracking information and re-index search api indexes https://asuntotuotanto.docker.so/fi/admin/config/search/search-api
- check some project where you can apply and if theres any then edit some project that you can apply 
   - https://asuntotuotanto.docker.so/fi/asuntohaku/haso
 - check some saleperson and use login link https://asuntotuotanto.docker.so/en/admin/people
 - login as salesperson and do applycation for some person on user listing page https://asuntotuotanto.docker.so/en/admin/people
 - check that you can do application for saleperson
 - now logout
 - login as suomi fi test account
 - now do application for suomi fi test account
